### PR TITLE
Hard-wire Buypass "directory" endpoints

### PR DESCRIPTION
**Pull request by szepeviktor**
I don't know how to solve it in code.

### CODE REVIEW

File: automatoes/acme.py [hunk 1]
```diff
@@ -237,7 +237,7 @@ def path(self, path):
 
 class AcmeV2(Acme):
 
-    def __init__(self, url, account, directory="directory", verify=None,
+    def __init__(self, url, account, directory="acme/directory", verify=None,
```
Comment (piraz) [inline automatoes/acme.py:240]: That will break compatibility the issue is on the url handling.

Comment (szepeviktor) [reply automatoes/acme.py:240]: I know, this is **not** a PR.

Comment (szepeviktor) [reply automatoes/acme.py:240]: How about adding a `--directory-uri` option instead of `--server`?

Comment (piraz) [reply automatoes/acme.py:240]: I do know and I got it. They are missing several keys on the responses, so they don't follow the rfc.

I'll trust the absolute urls and fix the relative ones, that's the key to fix. 

I've passed the url issue. I'll tweak other issues tonight and ship with 0.9.6 as I need to test if I don't break let's encrypt.

**Pull request closed without merging**